packages/big-design/src/components/TableNext/Actions/Actions.tsx

```diff
@@ -74,6 +75,7 @@ const InternalActions = <T extends TableItem>({
     <StyledFlex
       alignItems="center"
       aria-controls={tableId}
+      data-testid="table-controls"
```

Comment:
I think we want to avoid adding a test id wherever possible. I know we have some floating around the repo so we should try and clean those up eventually.

But this brings in the question, is this `Actions` component actually accessible? Do we need to add either a `toolbar` or `menubar` role to this element?